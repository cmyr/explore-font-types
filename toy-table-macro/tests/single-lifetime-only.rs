toy_table_macro::tables! {
    SingleLifetimeOnly<'a, 'b> {
        item_count: Uint24,
        #[count(item_count)]
        items: [Uint24],
    }
}

toy_table_macro::tables! {
    NoLifetimeBounds<'a: 'b> {
        item_count: Uint24,
        #[count(item_count)]
        items: [Uint24],
    }
}

fn main() {}
