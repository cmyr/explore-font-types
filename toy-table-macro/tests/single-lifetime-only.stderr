error: tables can contain at most a single lifetime
 --> tests/single-lifetime-only.rs:2:28
  |
2 |     SingleLifetimeOnly<'a, 'b> {
  |                            ^^

error: only a single unbounded lifetime is allowed
  --> tests/single-lifetime-only.rs:10:26
   |
10 |     NoLifetimeBounds<'a: 'b> {
   |                          ^^
